### DISCUSSION TITLE
SeaweedFS logo in SVG

### DISCUSSION THREAD

**Discussion by kvaps** [seaweedfs/seaweedfs#5824] [General]
Hi, we're going to adopt SeaweedFS in [Cozystack](https://cozystack.io) project for provisioning S3 backets.

I'd like to use the original logo in SVG format. Does anyone have vectorised logo for SeaweedFS?

**Comment by kvaps** [comment]
@chrislusf I saw you posted the original logo 9 years ago, do you still have sources?
https://github.com/seaweedfs/seaweedfs/blob/master/note/seaweedfs.png

**Reply by chrislusf** [reply]
Thanks! But that's the only version that I have.

**Comment by kvaps** [comment]
Okay, we're going to vectorize it, then I'll contribute it.
We have a designer @kvapsova. Would you like to consider another versions of logo (eg. monochromic one), or you want to keep it original?

**Reply by chrislusf** [reply]
Thanks, merged! 

Original one should be fine. I was thinking about a banner like logo (with height:width about 1:3 ratio, for example) for easier inline with text. Not sure if it is easy.
